utils/keys_utils.go

```diff
@@ -0,0 +1,166 @@
+package utils
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+	"log"
+)
+
+func Gen(id string) {
+	keyspath := GetPrjPath() + "worker/keys"
```

Comment:
Why you do not pass the key's path as a parameter or get it from a config file? Relying on that the project will always be stored in the subtree `"/src/github.com/ufcg-lsd/arrebol-pb-worker/"` does not look good. Also, you may wish to run the binary from another directory.